bokeh/_version.py

```diff
@@ -500,7 +500,7 @@ def get_versions():
         # versionfile_source is the relative path from the top of the source
         # tree (where the .git directory might live) to this file. Invert
         # this to find the root from __file__.
-        for i in cfg.versionfile_source.split('/'):  # lgtm [py/unused-loop-variable]
+        for _ in cfg.versionfile_source.split('/'):  # lgtm [py/unused-loop-variable]
```

Comment:
I suppose the `lgtm` comment can be removed now. It's interesting that lgtm complained here and not in the other cases. I'm wondering how reliable is that tool in general.

Reply:
Yeah i did think about that as well. Also, I don't get why it was used here in the first place

Reply:
@auirarrazaval can you remove the lgtm comment here so that this can be merged?

Reply:
@bryevdv  Done